### DISCUSSION TITLE
Compile for html5 issues

### DISCUSSION THREAD

**Discussion by Zyflx** [FunkinCrew/Funkin#1986] [Compiling help]
When I try to compile for html5 I get this:

source/WeekData.hx:11: characters 8-19 : You cannot access the sys package while targeting js (for sys.io.File)
source/editors/WeekEditorState.hx:34: characters 8-16 : ... referenced here
source/editors/MasterEditorMenu.hx:119: characters 37-52 : ... referenced here

Anyone know how to fix this?

**Comment by Snirozu** [comment]
``` haxe
#if sys
// sys refference here
#end
```
